digest/README.md

```diff
@@ -0,0 +1,254 @@
+# digest
+
```

Comment:
I am tempted to call this: drop_reason instead of digest. Then the opening statement here is:
This program demonstrates how to capture events from the datapath and logging them as events to the control plane. In this case we capture the reason why packets are dropped, what port they arrived on and what src mac address was observed on the packet. Packets could be dropped for two reasons:
1) the parser was unable to parse the packet
2) unable to find a matching entry on lookup of table nh_table.

<Rest of text below follows here>